docs/api.rst

```diff
@@ -4703,7 +4703,7 @@ Dealing with History Events
    :resjson string event_identifier: Common key. An event identifier grouping multiple events under a common group. This is how we group transaction events under a transaction, staking related events under block production etc.
    :resjson int sequence_index: Common key. This is an index that tries to provide the order of history entries for a single event_identifier.
    :resjson int timestamp: Common key. The timestamp of the entry
-   :resjson bool missing_accounting_rule: Common key. Present and `True` if the event is not processed by any accounting rule.
+   :resjson string event_accounting_rule_status: Common key. It states whether the event has a rule (``has rule``), will be processed because is affected by another event ``processed`` or it doesn't have any rule and won't be processed with other event (``not processed``).
```

Comment:
```suggestion
   :resjson string event_accounting_rule_status: Common key. It explains the status of accounting rules for the event. Possible values are: ``has rule``: Meaning the event has a rule. ``processed``: meaning the event will be processed because it is affected by another event. ``not processed`` meaning it doesn't have any rule and won't be processed by accounting.
```